caldate.h

```diff
@@ -0,0 +1,19 @@
+#ifndef CALDATE_H
+#define CALDATE_H
+
+struct caldate {
+  long year;
```

Comment:
I'm unsure about the "long" here. We clearly will not have this software in production when an int overflows, but this increases the struct size on 64 bit from 12 to 16 byte. This just fills a cacheline, so probably not really bad, but I'd like to understand why it is "long" and not "int".